doc/acl/Everflow-test-plan.md

```diff
@@ -14,28 +14,31 @@
 - [TODO](#todo)
 - [Open Questions](#open-questions)
 
-##Overview
+## Overview
 The purpose is to test functionality of Everflow on the SONIC switch DUT with and without LAGs configured, closely resembling production environment.
 The test assumes all necessary configuration, including Everflow session and ACL rules, LAG configuration and BGP routes, are already pre-configured on the SONIC switch before test runs.
 
-###Scope
+### Scope
 The test is targeting a running SONIC system with fully functioning configuration.
 The purpose of the test is not to test specific SAI API, but functional testing of Everflow on SONIC system, making sure that traffic flows correctly, according to BGP routes advertised by BGP peers of SONIC switch, and the LAG configuration.
 
 NOTE: Everflow+LAG test will be able to run **only** in the testbed specifically created for LAG.
 
-###Related **DUT** CLI commands
+### Related **DUT** CLI commands
 Manual Everflow configuration can be done using swssconfig utility in swss container.
 
     swssconfig <json-file to apply>
```

Comment:
latest test is now using `mirror_session.py` to create session and `acl-loader` with option `--session_name=` to apply rules

Reply:
Useing a script calling swsssdk to create mirror session does not look like a good idea. There is a better way, just use CLI command `config mirror_session add`.